package.json

```diff
@@ -42,17 +42,17 @@
   },
   "main": "dist/app/background.js",
   "dependencies": {
-    "@rancher/shell": "^0.1.2",
-    "css-loader": "4.3.0",
     "@kubernetes/client-node": "^0.17.1",
     "@nuxtjs/eslint-module": "^3.1.0",
     "@rancher/components": "0.1.0-alpha.5",
+    "@rancher/shell": "^0.1.2",
     "agent-base": "^6.0.2",
     "bufferutil": "^4.0.7",
     "cookie-universal-nuxt": "^2.2.2",
     "core-js": "^3.25.0",
     "cron-validator": "^1.3.1",
     "cronstrue": "^2.11.0",
+    "css-loader": "4.3.0",
```

Comment:
I know you didn't add this, but… shouldn't that be in `devDependencies`?

Reply:
I'm trying to keep dependencies in sync with what `@rancher/shell` requests. I agree that this should be safe to move to `devDependencies` and I will give it a try and another round of testing.

Reply:
No discernible issues with this change.